UPGRADE.md

```diff
@@ -2,6 +2,271 @@
 
 ## 2.0.0 (unreleased)
 
+### Database
+
+Dynamic Entity has been reduced to some basic fields. All previous data fields are merged into the `data` column.
+
+#### Data Migration
+Migrate the data fields into the json `data`. 
+```sql
+UPDATE
+  fo_dynamics as dyn
+SET
+  dyn.data = (
+    CONCAT(
+      '{',
+      SUBSTRING(dyn.data, 2, CHAR_LENGTH(dyn.data) -2),
+      IF(
+        STRCMP('', SUBSTRING(dyn.data, 2, LENGTH(dyn.data) -2)) = 0,
+        ' ',
+        ','
+      ),
+      IF(
+        dyn.salutation is not NULL,
+        CONCAT(
+          '\"salutation\":\"',
+          replace(dyn.salutation, '"', '\\\"'),
```

Comment:
what happens if a text includes `\`?

Reply:
Now I additionally escaped `\` and `/` are there any other symbols that should be escaped ?